apidoc/Titanium/UI/WebView.yml

```diff
@@ -530,6 +532,7 @@ events:
     properties:
       - name: url
         summary: The URL of the web document that is stopped.
+        type: String
     deprecated:
         since: "6.1.0"
```

Comment:
@appcelerator/android we should remove the code for this for 9.0.0 (and mark this property as removed in docs here), here's the code: https://github.com/appcelerator/titanium_mobile/blob/master/android/modules/ui/src/java/ti/modules/titanium/ui/widget/webview/TiWebViewClient.java#L152-L153